src/plays/personal-profile-card/components/profile-card.tsx

```diff
@@ -0,0 +1,114 @@
+import React from 'react'
+import {
+  AiOutlineFacebook,
+  AiOutlineGithub,
+  AiOutlineInstagram,
+  AiOutlineLinkedin,
+  AiOutlineTwitter,
+} from 'react-icons/ai'
+import ProfileType from '../types'
+
+interface Props {
+  profile: ProfileType
+}
+
+const profile = ({ profile }: Props) => {
```

Comment:
name of the component should start with capital letter

Reply:
Fixed.